### DISCUSSION TITLE
Advertising API,Sponsored Brands

### DISCUSSION THREAD

**Discussion by martin-git12** [amzn/ads-advanced-tools-docs#104] [Q&A]
Advertising API, how can the ASIN of Sponsored Brands corresponding to creative products in Reporting be obtained through the API?

**Comment by elizafarley** [comment]
Hi @martin-git12, there are two reports available to get information about ASIN-level performance. 

To get information on products purchased as a result of ads, you can use the [SB purchased product report](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v3/report-types#sponsored-brands). You can also use the the [SB ads report](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v2/report-types#ad-reports) to get performance details for advertised ASINs, as well as brand halo sales. 

Hope that answers your question.
